### PR TITLE
Add DevDocs for macOS

### PR DESCRIPTION
**Pull request by dteoh**
## Project URL

https://github.com/dteoh/devdocs-macos

## Category

Other

## Description

Added a link to DevDocs for macOS app.
 
## Why it should be included to `Awesome macOS open source applications ` (optional)

This app lets developers use [DevDocs](https://devdocs.io) as a standalone tool. It is a FOSS alternative to Dash.

## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in alphabetical order
- [x] Appropriate language icon(s) added if applicable
- [x] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by serhii-londar** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 10 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
3399 | [404](https://httpstatuses.com/404) | https://raw.githubusercontent.com/klauscfhq/ao/master/media/list-navigation.gif 
3400 | [404](https://httpstatuses.com/404) | https://raw.githubusercontent.com/klauscfhq/ao/master/media/compact-mode.png 
3402 | [404](https://httpstatuses.com/404) | https://raw.githubusercontent.com/klauscfhq/ao/master/media/ao-vibrant.png 
3403 | [404](https://httpstatuses.com/404) | https://raw.githubusercontent.com/klauscfhq/ao/master/media/black-theme.png 
3404 | [404](https://httpstatuses.com/404) | https://raw.githubusercontent.com/klauscfhq/ao/master/media/sepia-theme.png 
3405 | [404](https://httpstatuses.com/404) | https://raw.githubusercontent.com/klauscfhq/ao/master/media/custom-shortcut-keys.gif 
3406 | [404](https://httpstatuses.com/404) | https://raw.githubusercontent.com/klauscfhq/ao/master/media/dark-theme.png 
3407 | [404](https://httpstatuses.com/404) | https://raw.githubusercontent.com/klauscfhq/ao/master/media/scalable-interface.gif 
3409 | [404](https://httpstatuses.com/404) | https://raw.githubusercontent.com/klauscfhq/ao/master/media/logo.png 
5026 | Error | https://gielcobben.com/github/caption/github_cover.png <br> SSL_connect returned=1 errno=0 state=error: certificate verify failed


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by serhii-londar** [comment]
Hi @dteoh! Thanks for your contribution! We have new [Contribution Guidelines](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/CONTRIBUTING.md). Please read it and resubmit or edit your PR. Thank you!

**Comment by dteoh** [comment]
@serhii-londar Okay, updated.

File: applications.json [hunk 1]
```diff
@@ -3139,6 +3139,18 @@
...
     },
+    {
+      "repo_url" : "https://github.com/dteoh/devdocs-macos",
+      "title" : "DevDocs for macOS",
+      "screenshots" : [
+        "https://raw.githubusercontent.com/dteoh/devdocs-macos/master/img/screenshot.png"
+      ],
+      "short_description" : "An unofficial DevDocs API documentation viewer.",
+      "languages" : [
+        "swift_icon"
```
[the remark this answers is not in the document]
Comment (dteoh) [reply applications.json]: Fixed.

[the remark this answers is not in the document]
Comment (serhii-londar) [reply applications.json]: Thank you! 👍

**Pull request merged**